package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "freedom-social-quiver",
   "description": "Multi-server Social Provider",
-  "version": "0.0.2",
+  "version": "0.1.0",
```

Comment:
Might as well leave it at 0.0.3.  That's still a breaking API change in our semantic versioning system.